packages/components/src/SubHeaderBar/SubHeaderBar.component.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { Action } from '../Actions';
+import ActionBar from '../ActionBar';
+import InputTitleSubHeader from './InputTitleSubHeader';
+import theme from './SubHeaderBar.scss';
+
+function SubHeaderBarActions({ components, left, right, center, className }) {
+	return (
+		<div className={className}>
+			{components.map((component, index) => (
+				<ActionBar.Content
+					key={index}
+					tag={component.tag}
+					left={left}
+					center={center}
+					right={right}
+				>
+					{component.injectedComponent}
+				</ActionBar.Content>
+			))}
+		</div>
+	);
+}
+
+SubHeaderBarActions.propTypes = {
+	components: PropTypes.array.isRequired,
+	right: PropTypes.bool,
+	center: PropTypes.bool,
+	left: PropTypes.bool,
+	className: PropTypes.string,
+};
+
+function SubHeaderBar({ onGoBack, componentsCenter, componentsRight, className, ...rest }) {
+	const defaultComponentsLeft = [
+		{
+			injectedComponent: (
+				<Action
```

Comment:
Shoul have tooltip?

Reply:
I think it should have tooltip only if the trigger is on click
have both at the same time on hover make it look bad

Reply:
The tooltip is here. But it pop at the the top of the button, so not visible in the storybook.